### ISSUE TITLE
Mint failure - never completed after successful preview - wallet doesn't appear

### ISSUE THREAD

**Issue by psychedelicnature** [hicetnunc2000/hicetnunc#627]
**Mint failure - never completed after successful preview - wallet doesn't appear**
Everytime I try to mint a new work, clicking to mint on the preview screen, the screen goes blank forever, the wallet window never appears to confirm minting! 

**I expect the Temple wallet window to appear so I can confirm minting**
The wallet works normally for every other action such as buying new OBJKTs

**Screenshots**
![bug_inspect](https://user-images.githubusercontent.com/82925229/117731616-d2023b00-b1c4-11eb-8b4f-d2fcd0896909.JPG)

**Desktop:**
 - Windows 10
 - Browser Brave

**Additional context**
Wallet address : tz1iwXebhEReFRxM7n5DhRGdaBXaTGTEaG6C

**Comment by skenaja** [comment]
Please can you provide a screenshot from the Dev Tools console (right-click > inspect > console) to see if there are errors reported

**Comment by AmateurTemple** [comment]
don' know really how this works, I have the same issue and don't want to file again.

Mint failure after succesfull preview and fees spend.
Swap failure after succesfull interaction with wallet/tezos, and fees spend.
I've done those x5 each with succesfull spending of fees and fail of action.

**Comment by psychedelicnature** [comment]
> Please can you provide a screenshot from the Dev Tools console (right-click > inspect > console) to see if there are errors reported

Of course! Here it is
![bug_inspect_console](https://user-images.githubusercontent.com/82925229/116787135-d5851c00-aa78-11eb-9d94-d7b0cd5a9ade.JPG)

**Comment by andrevenancio** [comment]
> don' know really how this works, I have the same issue and don't want to file again.
> 
> Mint failure after succesfull preview and fees spend.
> Swap failure after succesfull interaction with wallet/tezos, and fees spend.
> I've done those x5 each with succesfull spending of fees and fail of action.

Please provide your wallet id so I can confirm what you're saying.

**Comment by vacuumn** [comment]
![1Capture](https://user-images.githubusercontent.com/1943043/117554003-c585b300-b009-11eb-8228-d7d89ba9e116.PNG)
Same here

Chrome, Win10

**Comment by aeonbloom** [comment]
Hi :) I've been stuck trying to mint for what seems like hours on the "preparing Objkt" screen. I have synced Temple wallet and have XTZ in there. Unsynced wallet and resynced to try again, it's not a large file. I have also tried multiple internet connections :)
![Screen Shot 2021-05-12 at 16 56 53](https://user-images.githubusercontent.com/84087730/117997308-2501fc80-b343-11eb-828b-d358f8969cb0.png)
![Screen Shot 2021-05-12 at 16 57 35](https://user-images.githubusercontent.com/84087730/117997397-35b27280-b343-11eb-8e00-2f2b8d88d1c8.png)
Macbook Pro
OS Version 10.13.6
Chrome
Wallet Address  -tz1e61Rw9QFDrufqZzXceShWbd747MWLwwbX
